### PR TITLE
fix(catalog-management): update examples after recent api changes

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary

Examples are aligned to the updated api. Worth to consider merge this PR after this one: https://github.com/IBM/platform-services-python-sdk/pull/118

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Andras-Csanyi** [comment]
@pyrooka Could you do your magic on this branch too?

File: examples/test_catalog_management_v1_examples.py [hunk 1]
```diff
@@ -0,0 +1,1622 @@
...
+from ibm_platform_services.catalog_management_v1 import *
+
+#
+# This file provides an example of how to use the Catalog Management service.
+#
+# The following configuration properties are assumed to be defined:
+# CATALOG_MANAGEMENT_URL=<service base url>
+# CATALOG_MANAGEMENT_AUTH_TYPE=iam
+# CATALOG_MANAGEMENT_APIKEY=<IAM apikey>
+# CATALOG_MANAGEMENT_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
```
Comment (pyrooka) [inline examples/test_catalog_management_v1_examples.py:33]: As I see there are more variables in use. Could you add those?

File: examples/test_catalog_management_v1_examples.py [hunk 2]
```diff
@@ -0,0 +1,1622 @@
...
+        try:
+            print('\nreplace_catalog() result:')
+            # begin-replace_catalog
+
+            catalog = self.catalog_management_service.replace_catalog(
+                catalog_identifier=catalog_id,
+                id=catalog_id,
+                tags=['python', 'sdk', 'updated'],
+                owning_account=account_id,
+                kind='vpe'
```
Comment (pyrooka) [inline examples/test_catalog_management_v1_examples.py:166]: If it's not a big deal please add trailing commas to every lines like this. Just like in the integration tests.
```suggestion
                kind='vpe',
```

**Review by pyrooka — APPROVED** [review]
All in all, it looks good.

**Pull request merged**